src/rgw/rgw_crypt.cc

```diff
@@ -35,8 +35,6 @@ using namespace rgw;
 /**
  * Encryption in CTR mode. offset is used as IV for each block.
  */
-#warning "TODO: move this code to auth/Crypto for others to reuse."
```

Comment:
@yehudasa Yeah, that may be true. But it's been annoying nowadays as it appears in every other PR builds.